### PR TITLE
add nvme to talos hw matrix

### PR DESCRIPTION
**Pull request by PrzemyslawBanasiak13**
Signed-off-by: pbanasiak <przemyslaw.banasiak@3mdeb.com>

### CODE REVIEW

File: docs/variants/talos_2/hardware-matrix.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ coreboot port on the Talos II platform.
 | **Network**                    | Local network wired connection                                |
 | **Attached devices**           | 1. SanDisk USB 3.2Gen1 16 GB                                  |
 |                                | 2. Logitech, Inc. Keyboard K120                               |
+|                                | 3. Micron 1100 256GB M.2 SATA3                                |
```
Comment (sulewskiprzemyslaw) [inline docs/variants/talos_2/hardware-matrix.md:19]: The `attached devices` list should be split as for any other Dasharo platforms. Take a look for an example this one: https://docs.dasharo.com/variants/protectli_vp46xx/hardware-matrix/

**Review by sulewskiprzemyslaw — CHANGES_REQUESTED** [review]

File: docs/variants/talos_2/hardware-matrix.md (hunk 1 above)
Comment (PrzemyslawBanasiak13) [reply docs/variants/talos_2/hardware-matrix.md:19]: Try pulling those changes and opening the preview of the `.md` file. In my case list is nicely split into rows.

Comment (PrzemyslawBanasiak13) [reply docs/variants/talos_2/hardware-matrix.md:19]: I've found a different mistake. SSD in other docs have their own category

Comment (sulewskiprzemyslaw) [reply docs/variants/talos_2/hardware-matrix.md]: List should be splited in meanings: another category for SSDs, another for attached USB devices, etc.

**Review by sulewskiprzemyslaw — APPROVED** [review]

**Pull request merged**